### ISSUE TITLE
Intermittently corrupted CSS keyframes when using comma

### ISSUE THREAD

**Issue by jaydenseric** [zeit/styled-jsx#432]
CSS keyframes are intermittently corrupted when using a comma to reuse frames.

#### Do you want to request a *feature* or report a *bug*?

Bug.

#### What is the current behavior?

If you reuse an animation frame in keyframes using a comma:

```jsx
const Animates = () => (
  <div>
    <style jsx>{`
      @keyframes test {
        0%,
        50% {
          opacity: 0;
        }
      }
      div {
        animation: test 1s;
      }
    `}</style>
  </div>
)
```

Then in Chrome on SSR the animation displays fine. Client-side navigate to a page that has the same animation and it still displays ok. Navigate to a page that does not have the animation and then to one that does: The animation no longer works. Strange `\u0` characters appear just after the commas, causing the keyframes to be rejected by the browser and the animation to fail:

<img width="120" alt="screen shot 2018-04-13 at 3 02 24 pm" src="https://user-images.githubusercontent.com/1754873/38717695-c52d555e-3f2b-11e8-8110-7cadc1256099.png">

In Firefox the animation never works, even just after SSR.

#### What is the expected behavior?

CSS animation keyframes should be generated correctly both in SSR and when mounting animated components in browser.

#### Environment (include versions)

* OS: macOS v10.13.4
* Browser: Chrome v65.0.3325.181, Firefox v59.0.2
* styled-jsx (version): v2.2.6

#### Did this work in previous versions?

Pretty sure. I'm returning to `styled-jsx` after using `styled-components` for a few months.

**Comment by jaydenseric** [comment]
@thysultan do you think this could relate to stylis?

**Comment by giuseppeg** [comment]
@jaydenseric maybe your editor or something in your setup (server?) is adding those. Can you try to remove all the indentation from your code and any space between `,` and `%50`?

**Comment by jaydenseric** [comment]
@giuseppeg in experimenting, this works:

```css
@keyframes test {
  0%, 50% {
    opacity: 0;
  }
}
```

And this doesn't:

```css
@keyframes test {
  0%,
  50% {
    opacity: 0;
  }
}
```

Which is unfortunate, since that's how Prettier formats CSS.

**Comment by thysultan** [comment]
@jaydenseric Sounds, related. Can you open an issue upstream.

**Comment by jaydenseric** [comment]
@giuseppeg has #455 been published? Can this be closed?

**Comment by giuseppeg** [comment]
👍  yes it is included in 2.2.7
